integ/run-integ.ps1

```diff
@@ -80,7 +80,7 @@ $env:AWS_FOR_FLUENT_BIT_CONTAINER_NAME = "aws-for-fluent-bit-$($env:TAG)"
 $env:ARCHITECTURE= "x86-64"
 $env:LOG_GROUP_NAME="fluent-bit-integ-test-$Platform-$env:ARCHITECTURE"
 $env:VOLUME_MOUNT_CONTAINER="C:/out"
-$env:ValidateS3Dockerfile = "Dockerfile.windows"
+$env:ValidateS3Dockerfile = "./scripts/dockerfiles/Dockerfile.windows"
```

Comment:
same comment as above

Reply:
My mistake. I didn't understand the context of this. Reverting